### ISSUE TITLE
[Library] [Design System Toolbox] Add missing illustration for tokens and missing declarations

### ISSUE THREAD

**Issue by ludovic35** [Orange-OpenSource/ouds-ios#1206]
### Description

Add illustration for :

- colors tokens (example borderBrandSecondary, borderBrandTertiary, ...)
- space maxWidth, minInteractinoArea
- ...

### Checklist for Core Team

- [x] Tokens have been updated (if needed)
- [x] Documentations have been updated (if needed)
- [x] Unit tests have been updated and still pass
- [x] UI tests have been updated and still pass
- [x] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by pylapp** [comment]
J'ai mis à jour ce qui concerne les tokens de couleurs car effectivement il manquait des déclarations et des affichages.

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765836505**
- Build tag: **9da812c**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765922908**
- Build tag: **8ba3d47**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/1.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **1.0.0**
- Build number: **1766065729**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_
